homeassistant/components/ads/__init__.py

```diff
@@ -210,3 +216,75 @@ def _device_notification_callback(self, notification, name):
             _LOGGER.warning("No callback available for this datatype")
 
         notification_item.callback(notification_item.name, value)
+
+
+class AdsEntity(Entity):
+    """Representation of ADS entity."""
+
+    def __init__(self, ads_hub, name, ads_var):
+        """Initialize ADS binary sensor."""
+        self._name = name
+        self._unique_id = ads_var
+        self._state_dict = dict()
+        self._state_dict[STATE_KEY_STATE] = None
+        self._ads_hub = ads_hub
+        self._ads_var = ads_var
+        self._event = None
+
+    async def async_initialize_device(self, ads_var, plctype,
```

Comment:
Just an advice for the future: It's nicer to break the line after the first parenthesis. There several reasons. Changing the function name won't impact the lines with the arguments. It's easier to read, since we read code from left to right and more code will be to the left.